### ISSUE TITLE
Training summary

### ISSUE THREAD

**Issue by ygjwd12345** [shekkizh/FCN.tensorflow#93]
I used your project to train.
my computer environment is win10 titan-xp  anaconda3 py3.5 tensorflow-gpu1.4
dataset is ADEChallengeData2016(download in support URL)
I train the net twice. First , batch=2,training time is about 7hours,but I found the result is so poor, the output almost is darkness. the loss is about 2
And I find in the issues ,that if we increase batch,the result maybe look perfect.So I try to exchange the batch=16(old issue recommend) and this training cost me about 20 hours, the train loss close to 0.25 but val loss is also about 2 but the output look good.
![image](https://user-images.githubusercontent.com/32895754/38120917-8c32d98c-33fe-11e8-96f1-12d2cad308d3.png)
I find a interesting thing,that the memory cost in batch 2 is more 2Gb than its in batch16,I can't understand.
